test/onnx.cpp

```diff
@@ -46,5 +52,45 @@ namespace menoh_impl {
             }
         }
 
+        TEST_F(ONNXTest, make_model_data_from_onnx_data_on_memory) {
+            std::ifstream input(model_filename, std::ios::binary);
+            std::istreambuf_iterator<char> begin(input);
+            std::istreambuf_iterator<char> end;
```

Comment:
Is the `end` variable initialized properly?
This might be trivial for C++ programmer but I'm not familiar with C++.

Reply:
Yes. It is initialized by its default constructor.

Reply:
Thanks!